src/test/java/academy/softserve/edu/asserts/AbstractElementAssert.java

```diff
@@ -309,6 +321,90 @@ public final AbstractElementAssert selectedDropdownEqualsIgnoreCase(final String
+            logFail("Element's {" + actual.getLocatorName() + "} data should be more than {" + quantity + "}");
+        } else {
+            logPass("Element's {" + actual.getLocatorName() + "} data more than {" + quantity + "}");
+        }
+        return this;
+    }
+
+    public final AbstractElementAssert isLessOrEqualsThan(int quantity) {
+        if (actual.getElements().size() > quantity) {
+
+            failWithMessage("Element's {%s} data should be less or equals, than {%s} ",
+                    actual.getLocatorName(), quantity);
+            logFail("Element's {" + actual.getLocatorName() + "} data should be less or equals" +
+                    " than {" + quantity + "}");
+        } else {
+            logPass("Element's {" + actual.getLocatorName() + "} data less or equals" +
+                    " than {" + quantity + "}");
+        }
+        return this;
+    }
+
+    public final AbstractElementAssert isEqualTo(int quantity) {
+        if (actual.getElements().size() != quantity) {
+
+            failWithMessage("Element's {%s} data should be equal {%s} ", actual.getLocatorName(), quantity);
+            logFail("Element's {" + actual.getLocatorName() + "} data should be equal {" + quantity + "}");
+        } else {
+            logPass("Element's {" + actual.getLocatorName() + "} data equal {" + quantity + "}");
+        }
+        return this;
+    }
+
+    public final AbstractElementAssert isParseIntEqualTo(int quantity) {
+        if (Integer.parseInt(actual.getText()) != quantity) {
+
+            failWithMessage("Element's {%s} data should be equal {%s} ", actual.getLocatorName(), quantity);
+            logFail("Element's {" + actual.getLocatorName() + "} data should be equal {" + quantity + "}");
+        } else {
+            logPass("Element's {" + actual.getLocatorName() + "} data equal {" + quantity + "}");
+        }
+        return this;
+    }
+
+    public final AbstractElementAssert isParseIntMoreThan(int quantity) {
+
+        if (Integer.parseInt(actual.getText()) <= quantity) {
+
+            failWithMessage("Element's {%s} data should be more, than {%s} ", actual.getLocatorName(), quantity);
+            logFail("Element's {" + actual.getLocatorName() + "} data should be more, than {" + quantity + "}");
+        } else {
+            logPass("Element's {" + actual.getLocatorName() + "} data more, than {" + quantity + "}");
+        }
+        return this;
+    }
+
+
+    public AbstractElementAssert isTrue(final boolean value) {
+
+        if (!value) {
+            failWithMessage("Element's {%s} Expected True, but in fact False!", actual.getLocatorName());
+            logFail("Element's {" + actual.getLocatorName() + "} Expected True, but in fact False! ");
+        } else {
+            logPass("Element's {" + actual.getLocatorName() + "} True");
+        }
+        return this;
+    }
+
+    public AbstractElementAssert isValueEqualTo(String someString) {
```

Comment:
STICK - we already have that -     public final AbstractElementAssert textEquals(final String condition)